api/v1beta1/disruption_types.go

```diff
@@ -124,17 +124,19 @@ type Reporting struct {
 	// It's expected to follow slack naming conventions https://api.slack.com/methods/conversations.create#naming or slack channel ID format
 	// +kubebuilder:validation:MaxLength=80
 	// +kubebuilder:validation:Pattern=(^[a-z0-9-_]+$)|(^C[A-Z0-9]+$)
-	// +kubebuilder:validation:Required
-	SlackChannel string `json:"slackChannel,omitempty" chaos_validate:"required"`
+	SlackChannel string `json:"slackChannel,omitempty"`
 	// Purpose determines contextual informations about the disruption
 	// a brief context to determines disruption goal
 	// +kubebuilder:validation:MinLength=10
-	// +kubebuilder:validation:Required
 	Purpose string `json:"purpose,omitempty"`
 	// MinNotificationType is the minimal notification type we want to receive informations for
 	// In order of importance it's Info, Success, Warning, Error
 	// Default level is considered Success, meaning all info will be ignored
 	MinNotificationType eventtypes.NotificationType `json:"minNotificationType,omitempty"`
+	// SlackUserEmail is the email of the user to send reporting information to
+	// +kubebuilder:validation:MaxLength=320
+	// +kubebuilder:validation:Pattern=(^[a-z0-9-_]+$)|(^C[A-Z0-9]+$)
+	SlackUserEmail string `json:"slackUserEmail,omitempty"`
```

Comment:
```suggestion
	SlackUserEmail string `json:"slackUserEmail,omitempty" chaos_validate:"email"`
```
I have some confidence this will work